### PR TITLE
Multiple email recipients and editable email templates

### PR DESCRIPTION
**Pull request by Nomafin**
Added an option to add multiple email recipients and define different templates (with the values of the fields usable in them) for different recipients.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "anttiviljami/wp-libre-form",
+  "name": "devgeniem/wp-libre-form",
```
Comment (anttiviljami) [inline composer.json]: I don't think I approve of this commit. :)

Comment (anttiviljami) [inline composer.json]: Can you switch branches and get rid of this commit so I can merge? :)

Comment (Nomafin) [inline composer.json]: You don't have to, I just had to quickly make this fork available to install via Composer and this was the fastest way to do so. There probably is better, but I'll change it back sooner or later. :)

Comment (anttiviljami) [inline composer.json]: Hehe, cheers! :)

File: inc/wplf-register-form.php [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+//             "with@defaulttemplate.com" => []
+//         ],
+//         "title_format" => "A submission from %email%",
+//         "thank_you" => "thank you html"
+//     ]);
+// }
+
+function register_libre_form( $name, $args ) {
+    // Throw an error if the function is not properly called
+    if ( ! post_type_exists( 'wplf-form' ) ) {
```
Comment (anttiviljami) [inline inc/wplf-register-form.php]: Just a small formatting thing, but really bothers me. :) Can you change the tabsize to 2 like all the other files, for this one as well?

**Comment by anttiviljami** [comment]
After some consideration, I've decided I'm not going to accept this implementation as is due to the user interface for adding and editing templates not being quite there yet. 

Feel free to fork or create a plugin for your implementation :)

I'll definitely work on your ideas later.

Cheers!

**Pull request closed without merging**